build.sbt

```diff
@@ -44,20 +51,22 @@ lazy val common = project
 
 lazy val backup = project
   .dependsOn(common)
-  .enablePlugins(RiffRaffArtifact, JavaAppPackaging)
+  .enablePlugins(RiffRaffArtifact, RiffRaffArtifact, JDebPackaging)
   .settings(standardSettings: _*)
   .settings(
     libraryDependencies ++= Seq(
       "com.typesafe.play" %% "play-logback" % "2.5.3",
       "com.microsoft.azure" % "azure-storage" % "3.1.0"
     ),
-    riffRaffPackageType := (packageZipTarball in Universal).value,
-    version := "1.0-SNAPSHOT"
+    riffRaffPackageType := (packageBin in Debian).value,
```

Comment:
I saw you hadn't modified the backup cloudformation script in the corresponding PR. I don't think you need to touch this one.